### PR TITLE
Automated Travis benchmark [ESD-1158]

### PR DESCRIPTION
**Pull request by pmiettinen**
This is done on top of https://github.com/swift-nav/libsbp/pull/656

And paired with https://github.com/swift-nav/piksi_tools/pull/985

### CODE REVIEW

**Comment by pmiettinen** [comment]
Test comment

**Comment by pmiettinen** [comment]
Test comment

**Comment by pmiettinen** [comment]
`travis.yml` based approach seems cumbersome, investigate more tox related approach.

**Comment by pmiettinen** [comment]
Working locally. Some tune up TODO for docker cooperation.

**Comment by pmiettinen** [comment]
Based on the runs this far, the performance ratio threshold could be set somewhere around 1.8.

File: python/sbp/jit/parse_float.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import cffi
```
Comment (silverjam) [inline python/sbp/jit/parse_float.py]: Prior to merging this we need to figure out how to get this module to generate itself automatically (for development), then build a compiled version for a PyPI wheel (this is optional, since it might be worthy of a whole separate PR to figure out to build and distribute this).

Comment (pmiettinen) [reply python/sbp/jit/parse_float.py]: How about a first phase remedy:

```
from sbp.jit import parse_float
from sbp.jit import parse_float_c
```

Also note the additions to `tox.ini` so another possibility would be to add the requirement to run `make python` before usage but that would probably go unnoticed in quite a lot of cases.

Comment (silverjam) [reply python/sbp/jit/parse_float.py]: @pmiettinen that might work, as long as the import succeeds, hopefully this doesn't require the user to have write access to wherever `sbp.jit.parse_float` is installed?

Comment (pmiettinen) [reply python/sbp/jit/parse_float.py]: Pretty sure the write access is required but I don't think this it matters at this phase as this solution is aimed only for development and the files shall reside under the repository? Do this now and later a better solution in a PR adding the `sbp.jit` package into the release PyPi package?

Comment (silverjam) [reply python/sbp/jit/parse_float.py]: Ok
